coral-component-actionbar/src/scripts/ActionBarSecondary.js

```diff
@@ -29,7 +29,7 @@ class ActionBarSecondary extends ActionBarContainer(BaseComponent(HTMLElement))
     let item = null;
     let wrappedItem = null;
 
-    for (let i = 0 ; i < this._itemsInPopover.length ; i++) {
```

Comment:
I am not sure how this solves the problem of ordering, can you provide more insights?

Reply:
Hi @CezCz, in this function, buttons are returned from popover to <coral-actionbar-secondary>. They are added after the "More button" one by one (line 43). It should be done in reversed order to keep buttons order which was before buttons were added to the popover.

Reply:
I see, but can you provide an example? I am trying to reproduce this issue on the coral-spectrum playground but I am not sure on how to follow. In the examples it seems the order is correct. I checked the internal jira tickets and it's not clear for me there either. 

This change might seem innocent, however if there are clients depending on the reverse ordering this will cause a regression on their side. Did you consider this? How do you think we should proceed here?

Reply:
@CezCz , 
**Steps to reproduce:**
1. Open https://opensource.adobe.com/coral-spectrum/examples/#actionbar
2. In second example, there is "more" button on right side. Click on it to open popover. "Search" button is on the first place in the popover. 
3. Close the popover and open it again.
**Actual behaviour:** "Search" button is on the last place. Buttons order is reversed.
**Expected behaviour:** "Search" button is still on the first place in the popover.

It also may cause another issue, if user has hidden buttons under "more" button, opens and closes "More" popover, then increases browser width to display hidden buttons from popover. Their order will be reversed.

I've checked "Coral/coralui-component-actionbar" repo. It's already fixed there in the same way as suggested in this PR. 
I think we should fix it as it's very unlikely that clients are dependent on changing buttons order and if somebody is then he might lock coral-spectrum version in package.json until fix it.